src/controllers/preferences.ts

```diff
@@ -27,7 +27,7 @@ export class PreferencesController extends EventEmitter {
     this.store = new ObservableStore(
       extensionStorage.getInitState({
         currentLocale: initLangCode || 'en',
-        idleOptions: { type: 'idle', interval: 0 },
+        idleOptions: { type: '1h', interval: 3600000 },
```

Comment:
It's much easier to understand what time value it is when written like this `60 * 60 * 1000` :-)